lib/saxerator/configuration.rb

```diff
@@ -5,7 +5,7 @@ class Configuration
 
     ALLOWED_OUTPUT_TYPES = {
       nokogiri: [:hash, :xml],
-      ox: [:hash]
+      ox: [:hash, :xml]
```

Comment:
@soulcutter I think we can just check that we have right output_type. I'll  update it.